### DISCUSSION TITLE
mobile feature: bluetooth keyboard paste

### DISCUSSION THREAD

**Discussion by LePenseur13** [logancyang/obsidian-copilot#1175] [Q&A]
**Is your feature request related to a problem? Please describe.**
I am working often on Obsidian from my Ipad Pro with an apple magic keyboard. 
When I want to paste something into the Chat with Copilot on the side pane, it never works. 
I have to use the touch screen to paste into the Chat with Copilot. 

**Describe the solution you'd like**
make that cmd+v and cmd+c (and STRG+v and STRG+c)
work in the copilot chat side panel on mobile

**Comment by logancyang** [comment]
hmm, why does a bluetooth keyboard on mobile have a different logic for copy and paste, and why is it something that can be fixed on the plugin level? Isn't this an OS thing?

**Comment by LePenseur13** [comment]
well it works in the notes in obsidian on mobile.

Not that you asked. it’s right. I cannot paste in any plugins that are in side panel view. 
also not the native obsidian search.

So it’s not an OS thing. But it’s also not directly related to your plugin. 

Maybe a workaround could be found. 

It used to be possible to open copilot in the notes area. Maybe it works then? But only 50% sure.
